# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/rene/dosemu/errno.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

diff -u misc.org/lpt.c misc/lpt.c
--- ./src/base/dev/misc.org/lpt.c	Sun Mar 11 14:27:24 2001
+++ ./src/base/dev/misc/lpt.c	Wed Mar 19 16:28:43 2003
@@ -17,6 +17,7 @@
 #include <unistd.h>
 #include <stdlib.h>
 #include <sys/wait.h>
+#include <errno.h>
 
 #include "emu.h"
 #include "dosio.h"
@@ -137,7 +138,6 @@
 printer_flush(int prnum)
 {
   int returnstat;
-  extern int errno;
 
   p_printf("LPT: flushing printer %d, %s\n", prnum, lpt[prnum].dev);
 
